### ISSUE TITLE
Show current state the process is in

### ISSUE THREAD

**Issue by nikita-fuchs** [aeternity/aepp-bridge#7]
Hey, the users need to know what the state the current process is in. 

After initiating the bridging, the info modal needs to read that 
1. The user must not close the window or the tokens will be lost 
2. "Transaction 1 of 2" 

After the bridging is successful, please display the information right on the site (ideally over the bridging UI) that the transfer was successful and what the user needs to do now.

When bridging from ETH to AE: 

"Congratulations, your tokens were successfully bridged ! {* Link to final transaction that minted the tokens maybe? *} To find them in your Superhero Wallet, open it up, select the account you bridged yout tokens to, click the "assets" tab and find the token listed in your list of tokens."

When bridging from AE to ETH:
"Congratulations, your tokens were successfully bridged ! To find them in your Wallet, open your lists of token assets" {* Link to final transaction that minted the tokens maybe? *}

**Comment by yusufseyrek** [comment]
Since, there is no need to claim any tokens on the other side of the bridge I don't think it's accurate to say "The user must not close the window or the tokens will be lost". And, approvals are not always required (when the address has allowance). But, I see your point, I am going to add a message at the end of the operation explaining that bridge action is completed. Also, going to attach the tx hash to the message.

**Comment by nikita-fuchs** [comment]
Yes absolutely, no reason for the old text, just an [ELI5](https://en.wiktionary.org/wiki/ELI5) explanation of what's going on and all info the user needs in order to check on the progress. **Ideally, that modal can't be closed by an outside click** which could have happened accidentally, hiding the necessary information, leaving the user in the dark.

**Comment by yusufseyrek** [comment]
Added a hard-to-close modal and collected the summary info of the bridge action.
